src/Filesystem/Filesystem.php

```diff
@@ -204,7 +204,7 @@ public function deleteDir(string $path): bool
 
         $result = true;
         foreach ($iterator as $fileInfo) {
-            $isWindowsLink = DS === '\\' && $fileInfo->getType() === 'link';
+            $isWindowsLink = DIRECTORY_SEPARATOR === '\\' && $fileInfo->getType() === 'link';
```

Comment:
Why does this need to use the full name?

Reply:
phpstan showed an error which I couldn't understand for this line. So tried the 1st thing that came to mind and it seemed to work :). Using PHP's constant is preferable either way.